### ISSUE TITLE
Error when running in batch norm mode

### ISSUE THREAD

**Issue by sahiliitm** [stevenpjg/ddpg-aigym#8]
Hi,

When I run the code in batch-norm mode for InvertedPendulum-v1 I get the [following error](http://pastebin.com/fP0vHNAU).
Any idea why this is happening?

Thanks
